src/main/java/com/labsynch/labseer/service/LotCorpName.java

```diff
@@ -0,0 +1,42 @@
+package com.labsynch.labseer.service;
+
+public class LotCorpName {
```

Comment:
I created this utility class as I needed a 3 element item and didn't want to hack it with something weird like an array or `HashMap<String, HashMap<integer,String>>`. Might be a little overkill but its a nice utility for this stuff.